www/og-image/utils/zodParams.ts

```diff
@@ -37,6 +37,14 @@ function truncateWords(str: string, maxWords: number) {
     : str;
 }
 
+export const fontParams = zodParams(
+  z.object({
+    family: z.string(),
+    weight: z.number().default(400),
```

Comment:
```suggestion
    weight: z.enum([400, ..., ...]).default(400),
```

Reply:
variable fonts could technically take any weight but yea we could restrict to the "default" ones